compiler/src/typed/typed_well_formedness.re

```diff
@@ -276,6 +277,29 @@ module WellFormednessArg: TypedtreeIter.IteratorArgument = {
             Grain_parsing.Location.prerr_warning(exp_loc, warning);
           };
         }
+      // Check: Warn if using Pervasives print on WasmXX types
+      | TExpApp(
```

Comment:
This should also be done for `toString`

Reply:
This now works on `toString`, I added stringification functions to the `DebugPrint` library as well so we could tell the user what to use.